package.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "poker-player-javascript",
```

Comment:
It looks like you've reindented this whole file. If it was intentional, maybe leave an [editorconfig](http://editorconfig.org/) settings file here so that the next person who edits it won't accidentally re-re-indent it with their own favourite editor settings. (BTW I like 2 spaces for JSON & JS too)